compiler/rustc_metadata/src/rmeta/mod.rs

```diff
@@ -404,6 +404,7 @@ define_tables! {
     proc_macro: Table<DefIndex, MacroKind>,
     module_reexports: Table<DefIndex, LazyArray<ModChild>>,
     deduced_param_attrs: Table<DefIndex, LazyArray<DeducedParamAttrs>>,
+    is_type_alias_impl_trait: Table<DefIndex, LazyValue<bool>>,
```

Comment:
```suggestion
    is_type_alias_impl_trait: Table<DefIndex, ()>,
```

we usually don't encode `false`, and just use the lack of an entry for that